src/xcode/ENA/ENA/Source/Client/HTTP Client/__tests__/HTTPClient+ExposureConfigTests.swift

```diff
@@ -26,6 +26,98 @@ final class HTTPClientExposureConfigTests: XCTestCase {
 
 	private let expectationsTimeout: TimeInterval = 2
 
+	#if INTEROP
+
+	func testInvalidEmptyExposureConfigurationResponseData() {
+		let stack = MockNetworkStack(
+			httpStatus: 200,
+			responseData: nil
+		)
+
+		let expectation = self.expectation(description: "HTTPClient should have failed.")
```

Comment:
For future tests you could consider setting [`assertforoverfulfill`](https://developer.apple.com/documentation/xctest/xctestexpectation/2806575-assertforoverfulfill) to true like this:

```swift
expectation.assertforoverfulfill = true
```

Then calling `fulfill` too frequently will cause the test to fail. I personally have also not done that consistently but I think this is a good habit to pick up. (Note to myself). :D

Reply:
I think its true by default when you create it with self.expectation(...)

Reply:
Oh right.